### PR TITLE
Add release instruction

### PR DESCRIPTION
**Pull request by marharyta**
# Checklist for author

## Documentation
- [x] Update docs/ (if applicable)

### CODE REVIEW

**Review by Macroz — APPROVED** [review]

File: docs/release.md [hunk 1]
```diff
@@ -46,28 +48,41 @@ Releases can be found on the github [release page](https://github.com/CSCfi/rems
...
-12. Press "Publish release".
+13. Press "Publish release".
 
 ## Updating dependencies
 
 After the release is done we want to update all the dependencies to the latest available versions.
+As per current suggestion, do it via:
+  - ```lein ancient```
+  - make sure you do not update dependencies that have no-update comment in ```project.clj```
+  - The suggested algorythms is:
```
Comment (Macroz) [inline docs/release.md:78]: I would phrase this (without typo) `The recommended procedure is:`

File: docs/release.md [hunk 2]
```diff
@@ -46,28 +48,41 @@ Releases can be found on the github [release page](https://github.com/CSCfi/rems
...
+    - ```clojure/clojurescript```
+    - backend
+    - frontend
+    - ```reframe/reagent```
+
+**note:** Be careful upgrading ```reagent``` and ```re-frame``` due to the extensive testing.
```
Comment (Macroz) [inline docs/release.md:84]: ... `due to extensive testing required!` ?

File: docs/release.md [hunk 3]
```diff
@@ -46,28 +48,41 @@ Releases can be found on the github [release page](https://github.com/CSCfi/rems
...
-11. Press "Attach binaries" to upload the `rems.jar` you got in step 8.
+12. Press "Attach binaries" to upload the `rems.jar` you got in step 8.
 
-12. Press "Publish release".
+13. Press "Publish release".
 
 ## Updating dependencies
 
 After the release is done we want to update all the dependencies to the latest available versions.
+As per current suggestion, do it via:
```
Comment (Macroz) [inline docs/release.md:75]: `You can do it via:`

File: docs/release.md (hunk 2 above)
Comment (marharyta) [reply docs/release.md:84]: good point

File: docs/release.md (hunk 3 above)
Comment (marharyta) [reply docs/release.md:75]: yay

File: docs/release.md (hunk 1 above)
Comment (marharyta) [reply docs/release.md:78]: yes, good point

**Review by opqdonut — APPROVED** [review]

**Pull request merged**